fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -89,6 +89,17 @@ object ItemKicker {
     case FreeHtmlKickerWithLink(body, _) => Some(body)
     case _ => None
   }
+
+  /** Return the canonical display text for a kicker */
+  def kickerDisplayText(itemKicker: ItemKicker): Option[String] = itemKicker match {
```

Comment:
@adamnfish Because the other function returns `HTML`, which I believe you do not want. We also do our own mappings on `frontend`, so we have no need for this new function.